packages/editor-ui/src/constants.ts

```diff
@@ -156,6 +156,9 @@ export const XERO_NODE_TYPE = 'n8n-nodes-base.xero';
 export const ZENDESK_NODE_TYPE = 'n8n-nodes-base.zendesk';
 export const ZENDESK_TRIGGER_NODE_TYPE = 'n8n-nodes-base.zendeskTrigger';
 
+export const CREDENTIAL_ONLY_NODE_PREFIX = 'n8n-creds-base';
+export const CREDENTIAL_ONLY_HTTP_NODE_VERSION = 4.1;
```

Comment:
can we just use the latest version when the http node is created?

Reply:
No, because the code in `utils/credentialOnlyNodes.ts` modifies the http node properties available in v4.1. If a new version of the http request node gets released, and has different properties, that code will break.

Instead the code will need to be updated when a new version of the http request node is released (but if it's not, nothing will break, credential nodes will keep working on 4.1)